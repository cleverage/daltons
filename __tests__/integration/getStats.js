const path = require('path')
const getStats = require('../../src/getStats')

describe('Stats parsing from CSV file', () => {
  it(`example csv`, async () => {
    expect(
      getStats(path.resolve(__dirname, '../../examples/simple/stats.csv')),
    ).toMatchSnapshot()
  }, 200)
})
