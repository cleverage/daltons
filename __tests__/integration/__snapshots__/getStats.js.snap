// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats parsing from CSV file example csv 1`] = `
Array [
  Object {
    "density": 1,
    "viewport": 1020,
    "views": 35,
  },
  Object {
    "density": 2,
    "viewport": 1021,
    "views": 3,
  },
  Object {
    "density": 1,
    "viewport": 1022,
    "views": 53,
  },
  Object {
    "density": 2,
    "viewport": 1023,
    "views": 12,
  },
  Object {
    "density": 1,
    "viewport": 1024,
    "views": 23,
  },
  Object {
    "density": 1.5,
    "viewport": 1025,
    "views": 27,
  },
  Object {
    "density": 1,
    "viewport": 1026,
    "views": 28,
  },
  Object {
    "density": 1.25,
    "viewport": 1027,
    "views": 32,
  },
  Object {
    "density": 1,
    "viewport": 1028,
    "views": 7,
  },
  Object {
    "density": 1,
    "viewport": 1029,
    "views": 18,
  },
  Object {
    "density": 1.1,
    "viewport": 1030,
    "views": 23,
  },
]
`;
